board/app/app_controller.php

```diff
@@ -11,14 +11,12 @@ function beforeFilter()
             'user/login'
         );
 
-        if (in_array(Param::get(DC_ACTION), $exclude)) 
-            {
-                return;
-            }
+    if (in_array(Param::get(DC_ACTION), $exclude)) {
+        return;
+    }
 
-         if (!is_logged_in()) 
-            {
-                redirect(url('user/login'));
-            }
+    if (!is_logged_in()) {
+        redirect(url('user/login'));
+        }
```

Comment:
Lines 14~20 are already in the correct indentions. Pls revert it back. These if statements are inside the function 'beforeFilter' right? So they should be indented inside this function block :wink: